NMS-13642: Add breaking changes for Twin (#3814)
OpenNMS/opennms: docs/modules/releasenotes/pages/whatsnew.adoc

RUNAS=root
----

=== Additional config needed when using Kafka for Minion Communication.

> Minion no longer uses OpenNMS Rest API to get Trapd Config updates. It uses Twin mechanism to sync with Trapd config.
> Minion needs additional config both on Minion and OpenNMS to enable Twin. Refer to deployment section from the docs.

Reviewer: ```suggestion
The Twin API has replaced the OpenNMS REST API for synchronizing Trapd configuration updates.
Additional settings are needed on both on Minion and {page-component-title} to enable Twin.

Refer to deployment section from the docs for details.
```

RUNAS=root
----

> === Additional config needed when using Kafka/Grpc for Minion Communication.

Reviewer 2: ```suggestion
=== Additional configuration needed when using Kafka/gRPC for Minion communication.
```

The Twin API has replaced the OpenNMS REST API for synchronizing Trapd configuration updates.

> Additional settings are needed on both on Minion and {page-component-title} to enable Twin.

Reviewer: ```suggestion
Additional settings are needed on both on Minion and {page-component-title} to enable Twin when not using the embedded ActiveMQ message broker.
```
